content/news/2022-08-02-release-22.08-highlights.md

```diff
@@ -0,0 +1,106 @@
++++
+title = "Release 22.08 Highlights"
+date = 2022-08-02T13:00:00Z
+type = "post"
+description = "Highlights of the 22.08 release."
+in_search_index = true
++++
+
+Helix is a modal text editor with built-in support for multiple selections,
+Language Server Protocol (LSP), tree-sitter, and experimental support for Debug
+Adapter Protocol (DAP).
+
+Today marks the 22.08 release, a release with plenty of fixes and new features.
+A big _thank you_ to our contributors! This release had 82 contributors. 🎉
+
+Let's check out the highlighted features for 22.08.
+
+## Indent guides
+
+{{ asciinema(id="indent-guides", width="94", height="25") }}
+
+Indent guides provide a visual representation for the current indentation
+level. Whether indent guides are rendered and what character to use as the
+guide are both configurable.
```

Comment:
Do we want to mention how to use indent-guides? It is not enabled by default.

Also, the current implementation does not look that nice. https://github.com/helix-editor/helix/issues/3298

Reply:
> Also, the current implementation does not look that nice

Yeah, we could hold up cutting the release until that issue is fixed. The easiest way to fix it would be to update all of those themes but that's not a long-term fix

Reply:
> The easiest way to fix it would be to update all of those themes but that's not a long-term fix

I think we definitely shouldn't be using incomplete solutions like that for a release.

Reply:
Looks like A-Walrus is already on the job 😀 https://github.com/helix-editor/helix/pull/3324

I'll add that to the 22.08 milestone so we get that in before cutting the release